docs/scanners.md

```diff
@@ -152,15 +152,15 @@ Scan rules:
 40016   Persistent XSS (Prime)
 40017   Persistent XSS (Spider)
 40018   SQL Injection
-40019   SQL Injection MySQL (Timing Based)
```

Comment:
Still inconsistent usage.

Reply:
? I made them all "Time Based" which is what was used in the PRs.

Reply:
Or do you mean the hyphens or lack thereof? (I didn't think these were the exact names.....I can do another pass and sync that if needed, but that just seems like a maintenance headache.)

Reply:
Sorry, we could have tackled that in the PRs. I didn't put the class naming together with the same issue.

I can fire in another, is `*TimeScanRule` or `*TimeBasedScanRule` better?